taskcluster/tc-build-utils.sh

```diff
@@ -315,3 +315,18 @@ do_nuget_build()
 
   nuget pack nupkg/deepspeech.nuspec
 }
+
+do_deepspeech_ios_framework_build()
+{
+  arch=$1
+  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so ${DS_DSDIR}/native_client/swift/libdeepspeech.so
+  cd ${DS_DSDIR}/native_client/swift
+  case $arch in
```

Comment:
nit: I think we could just have the case set a few variables like the SDK variant and have one `xcodebuild` call after:

```
  case $arch in
  "--x86_64")
    iosSdk="iphonesimulator"
    ;;
  "--arm64")
    iosSdk="iphoneos"
    ;;
  esac
xcodebuild -workspace deepspeech_ios.xcworkspace -scheme deepspeech_ios_test -configuration Release -arch ${arch} -sdk ${iosSdk} -derivedDataPath DerivedData CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO
```